addons/web_editor/static/src/xml/snippets.xml

```diff
@@ -174,12 +174,15 @@
                                     t-att-name="snippet.displayName"
                                     t-att-data-oe-snippet-id="snippet.id"
                                     t-att-data-module-id="snippet.moduleId"
-                                    t-on-click="_onSnippetClick"
-                                    data-tooltip="This block cannot be dropped anywhere on this page.">
+                                    t-on-click="_onSnippetClick">
                                     <t t-if="snippet.disabled">
                                         <img src="/web_editor/static/src/img/snippet_disabled.svg" class="o_snippet_undroppable"/>
                                     </t>
-                                    <div class="oe_snippet_thumbnail" t-att-class="{ 'o_we_already_dragging': snippet.renaming }" t-att-data-snippet="snippet.baseBody.dataset.snippet">
+                                    <div class="oe_snippet_thumbnail"
+                                        t-att-class="{ 'o_we_already_dragging': snippet.renaming }"
+                                        t-att-data-snippet="snippet.baseBody.dataset.snippet"
+                                        t-att-style="snippet.disabled ? 'pointer-events: auto; cursor: not-allowed;' : ''"
+                                        t-att-data-tooltip="snippet.disabled ? 'This block cannot be dropped anywhere on this page.' : ''">
```

Comment:
This need to fallback on `false`, not empty string, so qweb doesn't set the attribute.
Otherwise, it will just look like this when it's not disabled:
![image](https://github.com/odoo/odoo/assets/30048408/15daa367-9652-477c-89e7-d8247afb09df)